VK_V1.0/VK_V1.0/Servics/Operation/ParseData.swift

```diff
@@ -0,0 +1,26 @@
+// ParseData.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Парсинг
+final class ParseData: Operation {
```

Comment:
```suggestion
final class ParseData: Operation {
```

Он не универсальный, поэтому я бы подписала, что тут именно групп пирсинг